openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/DataQualityAndProfiler.spec.ts

```diff
@@ -464,7 +464,6 @@ test(
         await page.locator('#tableTestForm_displayName').clear();
         await page.fill('#tableTestForm_displayName', 'Updated display name');
         await page.click('.ant-modal-footer >> text=Submit');
-        await toastNotification(page, 'Test case updated successfully.');
```

Comment:
Why did we remove toast assertion from here?

Reply:
I think while saving, I mistakenly removed it and missed it.